src/components/app.test.js

```diff
@@ -1,26 +1,34 @@
-import React from 'react';
-import {shallow, mount} from 'enzyme';
-import App from './app';
-import BasicTutorial from './tutorials/basic-tutorial';
-import DeveloperPreview from './developer-preview';
+import { render } from 'react-dom'
+import React from 'react'
+import { MemoryRouter } from 'react-router-dom'
+import App from './app'
 
-it('renders without crashing', () => {
-  shallow(<App url={document.URL}/>);
-});
+it('displays default content', () => {
+  const div = document.createElement('div')
+  render((
+    <MemoryRouter>
+      <App />
+    </MemoryRouter>
+  ), div)
```

Comment:
@kencheeto tried mount and tests were failing, but I added back the `renders without crashing` test!